modules/cli/src/integrationTest/java/org/apache/ignite/rest/ItGeneratedRestClientTest.java

```diff
@@ -291,6 +296,20 @@ void nodeState() {
         });
     }
 
+    @Test
+    void logicalTopology() {
+        assertDoesNotThrow(() -> {
```

Comment:
`assertDoesNotThrow` seems redundant here as there is `assertThat` inside it, so if it throws, the test fails anyway